src/uikit.cs

```diff
@@ -18124,7 +18221,6 @@ partial interface UIMutableUserNotificationCategory {
 	}
 
 	[Deprecated (PlatformName.iOS, 10, 0, message: "Use 'UserNotifications.UNNotificationAction' instead.")]
-	[NoWatch]
```

Comment:
Why the removal?

Reply:
Because it was a partial class in another part of the same file. I merged them to be in a single place.